.travis.yml

```diff
@@ -72,23 +74,29 @@ matrix:
 
     # OSX {
 
-    - os: osx
-      osx_image: xcode9.4
-      env: >
-        TOOLCHAIN=osx-10-13-make-cxx14
-        PROJECT_DIR=examples/breakpad
-
-    - os: osx
-      osx_image: xcode9.4
-      env: >
-        TOOLCHAIN=osx-10-13-cxx14
-        PROJECT_DIR=examples/breakpad
-
-    - os: osx
-      osx_image: xcode9.4
-      env: >
-        TOOLCHAIN=ios-nocodesign-11-4-dep-9-3
-        PROJECT_DIR=examples/breakpad
+    # FIXME: OSX client build needs CPlusTest framework
+    # * https://travis-ci.com/FredrikAppelros/hunter/builds/109460348
+    #- os: osx
+    #  osx_image: xcode9.4
+    #  env: >
+    #    TOOLCHAIN=osx-10-13-make-cxx14
+    #    PROJECT_DIR=examples/breakpad
+
+    # FIXME: OSX client build needs CPlusTest framework
```

Comment:
I guess you can try to exclude tests?

Reply:
We could. I tried to fix the Android and iOS builds to no avail and eventually just gave up. I have no platform to try this out locally on.